packages/dashboard-frontend/src/store/Workspaces/index.ts

```diff
@@ -301,7 +248,7 @@ export const actionCreators: ActionCreators = {
 
   createWorkspaceFromDevfile:
     (
-      devfile: che.WorkspaceDevfile | devfileApi.Devfile,
+      devfile: devfileApi.Devfile,
       namespace: string | undefined,
       infrastructureNamespace: string | undefined,
```

Comment:
`namespace` and `infrastructureNamespace` are not used anymore and can be removed.